pyproject.toml

```diff
@@ -1,9 +1,18 @@
-
+[project]
+name = "visiumlint"
+dynamic = ["version"]
+description = "All of your favorite linters and formatters gathered in a single command."
+dependencies = ["black", "isort", "mypy", "pydocstyle", "pylint", "typer"]
 
 [build-system]
-requires = ["setuptools"]
+requires = ["setuptools>=45", "setuptools_scm[toml]>=6.2"]
 build-backend = "setuptools.build_meta"
 
+[project.scripts]
+visiumlint = "visiumlint.main:main"
+
+[tool.setuptools_scm]
```

Comment:
There is nothing in this section, what does it do?

Reply:
It indicates that we use the default settings of the setuptools_scm package.
CD does not run without this line.